packages/opc-back-to-top/src/opc-back-to-top.scss

```diff
@@ -9,7 +9,7 @@
 }
 
 .show {
-  font-family: "Red Hat Text", sans-serif;
+  font-family: 'RedHatText', sans-serif;
```

Comment:
Is it possible to use the variable here? It's a scss file right?

Reply:
Ok will change

Reply:
Done